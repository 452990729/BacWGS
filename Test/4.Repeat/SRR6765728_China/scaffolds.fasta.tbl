==================================================
file name: scaffolds.fasta          
sequences:          2258
total length:    1957732 bp  (1957732 bp excl N/X-runs)
GC level:         52.57 %
bases masked:      12492 bp ( 0.64 %)
==================================================
               number of      length   percentage
               elements*    occupied  of sequence
--------------------------------------------------
SINEs:                0            0 bp    0.00 %
      ALUs            0            0 bp    0.00 %
      MIRs            0            0 bp    0.00 %

LINEs:                0            0 bp    0.00 %
      LINE1           0            0 bp    0.00 %
      LINE2           0            0 bp    0.00 %
      L3/CR1          0            0 bp    0.00 %

LTR elements:         0            0 bp    0.00 %
      ERVL            0            0 bp    0.00 %
      ERVL-MaLRs      0            0 bp    0.00 %
      ERV_classI      0            0 bp    0.00 %
      ERV_classII     0            0 bp    0.00 %

DNA elements:         0            0 bp    0.00 %
     hAT-Charlie      0            0 bp    0.00 %
     TcMar-Tigger     0            0 bp    0.00 %

Unclassified:       177        13902 bp    0.71 %

Total interspersed repeats:    13902 bp    0.71 %


Small RNA:            0            0 bp    0.00 %

Satellites:           0            0 bp    0.00 %
Simple repeats:       0            0 bp    0.00 %
Low complexity:       0            0 bp    0.00 %
==================================================

* most repeats fragmented by insertions or deletions
  have been counted as one element
                                                      

The query species was assumed to be homo          
RepeatMasker Combined Database: Dfam_3.0
                                         
run with blastp version 3.0PE-AB [2009-10-30] [linux26-x64-I32LPF64 2009-11-17T18:52:53]
The query was compared to unclassified sequences in ".../RepBase.fasta"

