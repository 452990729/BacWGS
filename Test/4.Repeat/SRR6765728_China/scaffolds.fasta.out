   SW   perc perc perc  query                              position in query    matching          repeat          position in repeat
score   div. del. ins.  sequence                           begin end   (left)   repeat            class/family begin   end    (left)   ID

  234   10.3  0.0  2.6  NODE_1033_length_699_cov_2.453416    135   174  (525) C DNA-6-10_NV       Unspecified  (1239)   1165    1127   1  
  226   18.8  1.6  8.3  NODE_103_length_2322_cov_2.448610   1491  1554  (768) C Gypsy-2_CSa-I     Unspecified  (2721)   3000    2941   2  
  322   23.6  0.0  0.0  NODE_103_length_2322_cov_2.448610   1842  1913  (409) C tRNA-Tyr-TAC      Unspecified     (0)     76       5   3  
  259   21.2  0.0  5.6  NODE_103_length_2322_cov_2.448610   1847  1921  (401) + CryptonS-23_PI    Unspecified    2589   2659   (242)   4 *
  249   17.2  5.1  1.6  NODE_104_length_2314_cov_3.061974    267   325 (1989) + Mariner-4_LMi     Unspecified    1137   1197   (429)   5 *
  278   17.6  0.0  0.0  NODE_104_length_2314_cov_3.061974    278   328 (1986) + Mariner-1_PSo     Unspecified    1201   1251   (410)   6  
  657   21.0  0.6  0.6  NODE_1052_length_687_cov_3.292722     12   179  (508) + EnSpm-29_CCri     Unspecified    4541   4708  (6265)   7  
  230   32.8  0.0  5.4  NODE_106_length_2295_cov_1.877232   1822  1976  (319) + Sola1-2C_CCri     Unspecified    2813   2959  (9425)   8  
  235   15.2  9.4  7.7  NODE_1089_length_664_cov_2.546798    538   601   (63) + tRNA-SerCGA_OS    Unspecified      11     75     (7)   9  
  227    6.1  3.0  0.0  NODE_10_length_4419_cov_2.320577    3351  3383 (1036) + DNA2-25_CB        Unspecified     379    412   (574)  10  
  235   26.6  1.5  3.1  NODE_1107_length_657_cov_2.559801    369   434  (223) C BEL-615_AA-I      Unspecified  (4329)   1854    1790  11  
  312   17.5  0.0  0.0  NODE_1123_length_649_cov_2.373737    365   421  (228) C tRNA-Lys-AAA      Unspecified    (12)     64       8  12  
  242   27.6  0.0  7.5  NODE_1129_length_647_cov_2.331081     42   127  (520) C tRNA-Ser-TCA_     Unspecified     (0)     85       6  13  
  235   26.5  6.9  2.9  NODE_1169_length_625_cov_1.978947    282   382  (243) + hAT-10_XT         Unspecified      59    163 (10981)  14  
  225   20.0  0.0  0.0  NODE_1179_length_619_cov_1.998227     36    75  (544) C Copia-56_BD-I     Unspecified   (457)   4757    4718  15  
  225   27.4  2.7  0.0  NODE_1200_length_604_cov_2.167577    486   558   (46) C Gypsy-108_AA-I    Unspecified  (2703)   1916    1842  16  
  240   18.5 10.8  0.0  NODE_1223_length_588_cov_2.392120    464   528   (60) + DNA-7-5_NV        Unspecified     677    748  (2951)  17  
  225   19.7  2.9  2.9  NODE_1238_length_583_cov_1.916667     51   118  (465) C LINE1-N1I_OS      Unspecified   (978)    526     459  18  
  287   25.4  3.0  0.0  NODE_1277_length_568_cov_1.980507    146   212  (356) + tRNAIleATT_CB     Unspecified       3     71    (10)  19  
  227   22.9  0.0  0.0  NODE_1312_length_555_cov_1.766000    350   397  (158) + Jockey-N4_CQ      Unspecified    1233   1280   (513)  20  
  234   23.7  3.3  1.6  NODE_1328_length_545_cov_1.512245    272   331  (214) + ERV1-2-I_BT       Unspecified    5068   5128  (1158)  21  
  248   20.0  1.6  1.6  NODE_1346_length_531_cov_2.478992    434   494   (37) C Harbinger-2C_CPB  Unspecified  (4068)    444     384  22  
  227   19.2 14.3  1.1  NODE_134_length_2078_cov_2.001483    732   815 (1263) C SART-5_APi        Unspecified  (2409)   1996    1902  23  
  242   10.9  6.5  0.0  NODE_134_length_2078_cov_2.001483   1191  1236  (842) C L1-30_AAe         Unspecified  (2173)   3364    3316  24  
  226   27.4  3.2  0.0  NODE_1360_length_525_cov_1.902128     44   105  (420) C Zator-1N1_LCh     Unspecified   (651)    277     214  25  
  230   12.5  0.0  2.5  NODE_137_length_2063_cov_2.192231    605   645 (1418) + Polinton-2_TC     Unspecified   16912  16951    (30)  26  
  226   17.6 13.7  0.0  NODE_13_length_4332_cov_2.525836    3499  3549  (783) + BEL-15_AA-I       Unspecified    1888   1945  (3423)  27  
  255   23.7  0.0  4.2  NODE_1404_length_506_cov_2.230599    160   234  (272) C SINE_SM           Unspecified    (76)     85      14  28 *
  312   17.2  0.0  0.0  NODE_1404_length_506_cov_2.230599    173   230  (276) C tRNA-Lys-AAG      Unspecified    (11)     65       8  29  
  240   21.0 10.3  2.4  NODE_140_length_2055_cov_2.482500   1477  1554  (501) + Tad1-34_BG        Unspecified    5186   5269   (741)  30  
  230   21.8  6.5  6.5  NODE_1440_length_494_cov_2.369021    236   328  (166) C Copia-73_ZM-I     Unspecified  (2352)   3778    3686  31  
  225   18.8  2.1  0.0  NODE_1493_length_464_cov_2.017115    370   417   (47) C Copia-84_SB-I     Unspecified  (1712)   2805    2757  32  
  228   23.7  3.7  6.3  NODE_1550_length_438_cov_1.832898    169   249  (189) + Gypsy-9N1-I_XT    Unspecified     384    462  (2680)  33  
  245   19.6  0.0  0.0  NODE_1572_length_430_cov_1.584000    283   333   (97) + Gypsy-10_DVir-LTR Unspecified    1962   2012    (26)  34  
  245   13.2  7.5  0.0  NODE_1619_length_416_cov_1.418283    142   194  (222) + Jockey-4_DK       Unspecified    2694   2750  (1371)  35  
  228    8.1  0.0  5.4  NODE_1664_length_399_cov_2.796512    330   368   (31) + Copia-91_VV-I     Unspecified    3680   3716   (904)  36  
  234   11.2  0.0 11.1  NODE_167_length_1901_cov_2.264355    898   947  (954) C Gypsy-28_PSo-I    Unspecified  (3870)   2583    2539  37  
  234   14.3  0.0  0.0  NODE_169_length_1894_cov_2.008700   1451  1485  (409) C Gypsy-13_OD-I     Unspecified  (3810)   3383    3349  38  
  226   10.8  2.7  0.0  NODE_178_length_1847_cov_2.068080    417   453 (1394) + Gypsy-258_AA-I    Unspecified    2996   3033  (2219)  39  
  325   21.0  0.0  0.0  NODE_1830_length_341_cov_1.416084    132   193  (148) + tRNA-Tyr-TAC      Unspecified       6     67     (9)  40  
  228   25.9  8.4 10.8  NODE_1834_length_338_cov_2.724382    142   236  (102) C Gypsy-71H_CCri-I  Unspecified  (7980)   3148    3056  41  
  238   13.5  1.9  3.8  NODE_184_length_1806_cov_2.535694      1    54 (1752) + piggyBacX-20_CCri Unspecified    1484   1536  (3736)  42  
  229   18.2  0.0  0.0  NODE_1859_length_333_cov_3.111511    185   228  (105) + GYZMA1_I          Unspecified    3072   3115  (1695)  43  
  232   14.5 12.5  1.6  NODE_185_length_1805_cov_2.230286    630   685 (1120) C LTR23_Tbel        Unspecified   (221)    271     210  44  
  233   24.1  7.4  0.0  NODE_1944_length_304_cov_1.903614    219   272   (32) C EnSpm-11_ZM       Unspecified   (110)   7816    7759  45  
  241   12.5  4.1  2.0  NODE_1971_length_297_cov_1.983471    227   275   (22) + Gypsy-15_CQ-I     Unspecified    2275   2324  (2932)  46  
 2642    0.3  0.0  0.0  NODE_1981_length_293_cov_2.109244      1   293    (0) C L1HS_5end         Unspecified   (908)   1228     936  47  
  225   15.9  4.5  0.0  NODE_1994_length_289_cov_2.525641    143   186  (103) + Gypsy6-I_VC       Unspecified    3533   3578  (1352)  48  
  232   11.8  7.5  3.6  NODE_199_length_1748_cov_2.886592   1024  1076  (672) + P-1_Dpulex        Unspecified    4330   4384   (323)  49  
  228   26.4  0.0  3.8  NODE_1_length_8085_cov_2.795143     4788  4842 (3243) + BEL-88_CQ-I       Unspecified    5238   5290   (483)  50  
  226   20.0 10.8  0.0  NODE_1_length_8085_cov_2.795143     6574  6638 (1447) C BEL-599_AA-I      Unspecified  (4832)    606     535  51  
  236   17.4  9.0  5.8  NODE_2018_length_280_cov_2.422222     85   151  (129) + Gypsy-12_Sit-I    Unspecified    2605   2673  (1729)  52  
  240   14.3  0.0  0.0  NODE_201_length_1745_cov_2.020710   1649  1683   (62) C Gypsy-35_AIp-I    Unspecified  (4389)    115      81  53  
  242   19.7  2.2 10.7  NODE_2023_length_279_cov_2.245536     95   185   (94) + Mariner-N1B_LMi   Unspecified     186    269   (657)  54 *
  307   26.6  0.0  1.3  NODE_2023_length_279_cov_2.245536     99   178  (101) + tRNAThr_CB        Unspecified       6     84     (0)  55  
  235   16.1  3.1  4.7  NODE_2023_length_279_cov_2.245536    206   270    (9) + SINE2-2_SP        Unspecified      10     73   (178)  56  
  238    2.8  0.0 18.9  NODE_2033_length_276_cov_2.289593     18    61  (215) + LINE1-31_ZM       Unspecified    1047   1083  (2599)  57  
  230   26.4  4.2  0.0  NODE_204_length_1734_cov_2.550923    752   823  (911) + Jockey-2_DGri     Unspecified     664    738  (5294)  58 *
  241   21.0  0.0  0.0  NODE_204_length_1734_cov_2.550923    762   823  (911) + Ag-Jock-1         Unspecified    1403   1464  (3261)  59  
  250   19.0  6.8  4.4  NODE_206_length_1727_cov_1.804426    527   614 (1113) + SINE2-2_SP        Unspecified       6     95   (156)  60 *
  266   19.7  0.0  3.3  NODE_206_length_1727_cov_1.804426    531   593 (1134) + SINE2-1_VC        Unspecified       6     66   (602)  61  
  242   15.4  0.0  0.0  NODE_210_length_1701_cov_1.963548   1135  1173  (528) + Polinton-1N1_DR   Unspecified    7565   7603  (7866)  62  
 1551    0.6  0.0  0.0  NODE_2185_length_225_cov_3.247059     47   225    (0) C L1HS_5end         Unspecified   (737)   1399    1221  63  
  246    4.0  0.0  0.0  NODE_2256_length_62_cov_1.000000       5    54    (8) + Polinton-1_CB     Unspecified    6868   6917  (9716)  64  
  246    4.0  0.0  0.0  NODE_2256_length_62_cov_1.000000      12    61    (1) + Polinton-1_CB     Unspecified    6868   6917  (9716)  65  
  242   21.9  0.0  7.3  NODE_22_length_3984_cov_3.080173     267   325 (3659) C Gypsy-1_CDC-I     Unspecified  (1152)   4213    4159  66  
  322   20.4  6.0  1.9  NODE_22_length_3984_cov_3.080173    1503  1602 (2382) + CACTA-N           Unspecified     585    688   (299)  67  
  230   21.1  1.9  0.0  NODE_235_length_1631_cov_1.773477   1139  1190  (441) C Tad1-1_BG         Unspecified   (286)   6050    5998  68  
  244   21.7  1.6  6.6  NODE_236_length_1628_cov_2.108709   1014  1077  (551) C Harbinger-17_CCri Unspecified   (495)   2269    2209  69  
  603   32.4  2.0  2.0  NODE_236_length_1628_cov_2.108709   1374  1628    (0) + EnSpm-29_CCri     Unspecified    4269   4523  (6450)  70  
  236   12.2 12.2  0.0  NODE_253_length_1588_cov_2.770385   1542  1582    (6) + Copia-75_AA-I     Unspecified    1171   1216  (3142)  71  
  227   27.8  1.4  1.4  NODE_26_length_3834_cov_2.571315    2730  2802 (1032) C Copia3-I_CR       Unspecified  (1450)   3890    3818  72  
  235    3.6  0.0  0.0  NODE_279_length_1527_cov_1.894022     33    60 (1467) + Copia-129_SB-I    Unspecified    1355   1382  (4271)  73  
  225   27.9  4.3  1.4  NODE_285_length_1517_cov_1.978112    366   434 (1083) C LTR-1_SBi         Unspecified   (285)   4453    4383  74  
  238   15.9  0.0  2.3  NODE_291_length_1504_cov_2.245687     53    97 (1407) C BEL30-I_DR        Unspecified  (4646)   2002    1959  75  
  237   22.2  0.0  0.0  NODE_296_length_1493_cov_1.995132   1365  1409   (84) + ERV1-4_TSy-I      Unspecified    4263   4307   (807)  76  
  243   25.9  0.0  0.0  NODE_29_length_3746_cov_1.964779     833   886 (2860) + Copia-27_PIT-I    Unspecified    2733   2786  (2231)  77  
  256   23.7  1.7  0.0  NODE_304_length_1482_cov_2.184303    472   530  (952) C Gypsy-3_CFl-I     Unspecified  (3706)   1746    1687  78  
  230   29.9  4.6  0.0  NODE_304_length_1482_cov_2.184303   1010  1096  (386) C Gypsy-73_MN-I     Unspecified  (1223)   5516    5426  79  
  239   24.6  0.0  1.8  NODE_309_length_1472_cov_2.315455    196   253 (1219) C Gypsy-15_SCH-I    Unspecified  (6626)   4663    4607  80  
  231   24.2  0.0  0.0  NODE_30_length_3717_cov_2.176133     768   829 (2888) C BivaV-SINE3_CoAn  Unspecified   (206)     64       3  81  
  240   32.9  0.0  1.4  NODE_30_length_3717_cov_2.176133     860   933 (2784) C tRNA-Arg2_OS      Unspecified     (0)     73       1  82  
  233    6.7  0.0  0.0  NODE_30_length_3717_cov_2.176133     967   996 (2721) C FEILAI-1B_AAe     Unspecified   (210)     79      50  83  
  230   14.7  0.0  2.4  NODE_31_length_3697_cov_2.760297    2893  2934  (763) C Copia-1_ALY-I     Unspecified  (4573)    225     185  84  
  251   22.2  1.3  4.1  NODE_31_length_3697_cov_2.760297    3224  3298  (399) + Mariner3-4_LMi    Unspecified     172    244  (2787)  85 *
  256   23.8  0.0  3.2  NODE_31_length_3697_cov_2.760297    3234  3298  (399) C TE-3_LVa          Unspecified  (1778)    882     820  86  
  239   17.4  2.2  0.0  NODE_320_length_1439_cov_2.726156    942   987  (452) + Helitron-1_AEc    Unspecified    5637   5683  (2124)  87  
  284   26.2  6.9  2.8  NODE_338_length_1390_cov_2.560300   1229  1330   (60) C GarnAlu3B         Unspecified   (147)    113       8  88  
  248   24.1  1.8  1.8  NODE_353_length_1367_cov_2.041921    503   557  (810) C Gypsy-160_AA-I    Unspecified  (2493)   3594    3540  89  
  227   13.1  2.1  2.1  NODE_354_length_1366_cov_2.196034   1000  1046  (320) + Penelope-1_DR     Unspecified     524    570  (3876)  90  
  226   15.0  7.5  0.0  NODE_369_length_1343_cov_1.837733   1238  1277   (66) C DNA5-1_PSt        Unspecified    (44)    377     335  91  
  250   30.2  3.6  7.6  NODE_372_length_1332_cov_2.119029    472   718  (614) + CRE-15_CCri       Unspecified    2090   2327   (121)  92  
  231   15.6  4.4  6.0  NODE_38_length_3490_cov_2.610189    1289  1356 (2134) C Tx1-4_DR          Unspecified  (2213)   3322    3256  93  
  236   26.7  5.3  0.0  NODE_38_length_3490_cov_2.610189    1801  1875 (1615) + Ogre-PT2_LTR      Unspecified    2017   2095   (680)  94  
  229    7.9  0.0  5.3  NODE_38_length_3490_cov_2.610189    2060  2099 (1391) C Copia-6_ALY-I     Unspecified  (1436)   2620    2583  95  
  249   15.9 14.3  0.0  NODE_38_length_3490_cov_2.610189    2147  2209 (1281) + RTE-7_LCh         Unspecified    2532   2603  (1403)  96  
  230   17.0  6.2  2.0  NODE_392_length_1309_cov_1.971292    449   496  (813) + Copia-7_PGr-I     Unspecified    1154   1203  (3759)  97  
  227   31.1  2.2  6.8  NODE_392_length_1309_cov_1.971292    511   648  (661) C Jockey-3_DGri     Unspecified  (1212)   3501    3370  98 *
  261   30.3  0.0  2.8  NODE_392_length_1309_cov_1.971292    538   649  (660) + R1_DYa            Unspecified     721    829  (4684)  99  
  253   11.4  0.0  0.0  NODE_395_length_1306_cov_2.329337    546   580  (726) C Gypsy-69_PAb-I    Unspecified  (4037)    913     879 100  
  229   20.8  8.3  0.0  NODE_396_length_1306_cov_2.146283    378   425  (881) C Nimb-31_LMi       Unspecified  (1066)   4875    4824 101  
  230   24.0  0.0  0.0  NODE_403_length_1297_cov_2.557971   1166  1215   (82) + hATm-41_HM        Unspecified     185    234  (2750) 102  
  231   18.0  1.9  3.9  NODE_412_length_1290_cov_2.336842    253   304  (986) C hAT-42_Lch        Unspecified  (1459)   2016    1966 103  
  230    6.7 10.4  6.0  NODE_430_length_1259_cov_2.241694     68   115 (1144) C R7Ag2             Unspecified  (3780)   2810    2761 104  
  225   19.6  8.1  7.0  NODE_477_length_1180_cov_2.351111    864   962  (218) C P-30_HM           Unspecified  (3958)   2315    2216 105  
  239   20.5 13.5  1.2  NODE_48_length_3180_cov_3.223040    1448  1521 (1659) + Helitron-1_SP     Unspecified    8027   8109  (1991) 106  
  225    9.3 13.6  2.0  NODE_490_length_1165_cov_2.574775    182   225  (940) C Gypsy-1_PGr-I     Unspecified  (2056)   5764    5716 107  
  240   22.4  2.0  0.0  NODE_490_length_1165_cov_2.574775    490   538  (627) C Gypsy-11_CFl-LTR  Unspecified   (909)    306     257 108  
  242   22.4  0.0  1.7  NODE_492_length_1165_cov_1.801802    606   664  (501) + Copia-39_AIp-I    Unspecified     222    279  (4631) 109  
  230   17.9  1.7  3.5  NODE_499_length_1154_cov_2.083712     47   104 (1050) + Gypsy-15_ATr-I    Unspecified   11118  11174  (2772) 110  
  316   21.6 15.4  3.8  NODE_4_length_7157_cov_3.324838     4252  4511 (2646) + hAT-4_RC          Unspecified     938   1226  (3888) 111 *
  436   28.8  1.5  4.5  NODE_4_length_7157_cov_3.324838     4265  4468 (2689) C Gypsy-32_ADe-I    Unspecified  (9476)   2053    1856 112  
  385   29.7  0.0  3.5  NODE_4_length_7157_cov_3.324838     4282  4515 (2642) C MarinerL-1_CR     Unspecified (14376)   1612    1387 113 *
  453   23.2  0.0 10.8  NODE_4_length_7157_cov_3.324838     4328  4511 (2646) C Gypsy-32_ADe-I    Unspecified  (9464)   2065    1900 114  
  392   23.2  0.0  0.0  NODE_501_length_1153_cov_2.651184    460   541  (612) C ALPINE2           Unspecified    (56)     88       7 115  
  245   20.7  0.0  1.7  NODE_501_length_1153_cov_2.651184    562   620  (533) C tRNA-Lys-AAA      Unspecified    (11)     65       8 116  
  239   17.6  0.0  3.5  NODE_501_length_1153_cov_2.651184    569   627  (526) C SINE2-7_SP        Unspecified   (149)     62       6 117 *
  231   17.4  2.2  0.0  NODE_508_length_1148_cov_2.735590    674   719  (429) + Copia-92_SB-I     Unspecified    4037   4083   (308) 118  
  230   16.7  2.0  4.1  NODE_523_length_1130_cov_2.275349    416   465  (665) + Gypsy65-ZM_I      Unspecified    5053   5101   (512) 119  
  229   13.2  0.0 17.0  NODE_525_length_1129_cov_2.257914    128   182  (947) + Gypsy5-VV_I       Unspecified    4384   4430   (132) 120  
  269   12.5  5.4  0.0  NODE_53_length_3084_cov_2.179597      28    83 (3001) + Gypsy-91_ZM-I     Unspecified      15     73  (4549) 121  
  234   14.6  5.7 12.1  NODE_544_length_1096_cov_2.370797    111   180  (916) C Harbinger-2_XT    Unspecified  (2137)  10669   10604 122  
  233   14.3  0.0  2.4  NODE_593_length_1039_cov_2.147358    369   411  (628) + ATCopia34_I       Unspecified     658    699  (3561) 123  
  235   17.5  8.1  8.1  NODE_599_length_1036_cov_2.194699    303   364  (672) + Helitron-1_PTrit  Unspecified    4112   4173   (257) 124  
  225   32.0  0.0  0.0  NODE_5_length_6840_cov_3.507148     2029  2128 (4712) + SSU-rRNA_Lma      Unspecified    1163   1262   (875) 125 *
  661   32.2  7.9  3.2  NODE_5_length_6840_cov_3.507148     2068  2715 (4125) + SSU-rRNA_Tps      Unspecified     981   1657   (147) 126  
  357   28.4  5.5 11.4  NODE_5_length_6840_cov_3.507148     2457  2837 (4003) + SSU-rRNA_Giardia  Unspecified    1092   1452    (12) 127 *
  266   27.4  3.5  1.1  NODE_5_length_6840_cov_3.507148     2943  3027 (3813) + SINE2-2_SP        Unspecified       2     88   (163) 128 *
  296   23.7  0.0  0.0  NODE_5_length_6840_cov_3.507148     2953  3011 (3829) + tRNA-Asn_OS       Unspecified       8     66    (14) 129  
  345   24.2  0.0  1.1  NODE_5_length_6840_cov_3.507148     3028  3119 (3721) + ALPINE1           Unspecified       4     94    (33) 130  
  252   27.1  5.0  5.0  NODE_5_length_6840_cov_3.507148     3035  3135 (3705) + SINE2-1_PL        Unspecified      13    113   (135) 131 *
  304   25.3  0.0  1.1  NODE_5_length_6840_cov_3.507148     3619  3710 (3130) + LSU-rRNA_Hca      Unspecified      35    125  (3530) 132 *
  321   24.1  0.0  0.0  NODE_5_length_6840_cov_3.507148     3631  3717 (3123) + LSU-rRNA_Ddi      Unspecified      23    109  (3132) 133  
  315   20.0  4.5  3.4  NODE_5_length_6840_cov_3.507148     3848  3935 (2905) + LSU-rRNA_Ddi      Unspecified     293    381  (2860) 134 *
  390   24.0  0.0  0.0  NODE_5_length_6840_cov_3.507148     3870  3965 (2875) + LSU-rRNA_Hca      Unspecified     338    433  (3222) 135  
  292   24.5  1.0  3.0  NODE_5_length_6840_cov_3.507148     3893  3993 (2847) + LSU-rRNA_Ldo      Unspecified      11    109   (881) 136 *
  264   29.8  3.2  2.6  NODE_5_length_6840_cov_3.507148     4116  4270 (2570) + LSU-rRNA_Giardia  Unspecified     518    673  (2017) 137  
 1043   22.6  3.1  4.4  NODE_5_length_6840_cov_3.507148     4402  4725 (2115) C Copia-18_BD-I     Unspecified    (64)   4768    4449 138  
  279   27.4  4.9 14.8  NODE_5_length_6840_cov_3.507148     5047  5290 (1550) + LSU-rRNA_Giardia  Unspecified    1472   1694   (996) 139 *
  713   25.4  2.7  5.1  NODE_5_length_6840_cov_3.507148     5214  5433 (1407) + LSU-rRNA_Giardia  Unspecified    1616   1830   (860) 140  
  475   28.9  6.2  8.7  NODE_5_length_6840_cov_3.507148     5217  5568 (1272) + LSU-rRNA_Hca      Unspecified    2411   2754   (901) 141 *
  567   27.7  6.4  6.4  NODE_5_length_6840_cov_3.507148     5645  6127  (713) + LSU-rRNA_Hsa      Unspecified    4123   4605   (430) 142 *
  598   30.0  4.5  4.5  NODE_5_length_6840_cov_3.507148     5649  6046  (794) + LSU-rRNA_Dme      Unspecified    3042   3439   (461) 143  
  492   31.6  4.3  4.7  NODE_5_length_6840_cov_3.507148     5656  6210  (630) + LSU-rRNA_Cel      Unspecified    2699   3251   (258) 144 *
  228   22.2  1.8  3.6  NODE_603_length_1033_cov_1.829243    746   801  (232) C BEL-20_AG-I       Unspecified  (1882)   3429    3375 145  
  230   10.5  0.0  2.6  NODE_613_length_1023_cov_1.960744    852   890  (133) + P-22_HM           Unspecified    3093   3130    (35) 146  
  225   22.0  0.0  4.0  NODE_63_length_2833_cov_2.200504    1618  1669 (1164) + Loa-1_SIn         Unspecified    4261   4310   (816) 147  
  476   32.4  1.4  2.1  NODE_694_length_939_cov_2.038462     653   939    (0) C MuDR-3_CCri       Unspecified  (6710)   2514    2230 148  
  230   14.3  0.0  2.9  NODE_716_length_916_cov_1.825784     499   534  (382) + BEL-74_CQ-I       Unspecified     312    346  (5823) 149  
  227   15.1  9.3  1.7  NODE_736_length_899_cov_2.248815      40    93  (806) + BEL-626_AA-I      Unspecified    6179   6236   (673) 150  
  235   19.6 12.3  1.6  NODE_741_length_894_cov_2.476758     787   843   (51) C R1-1_DT           Unspecified   (164)   4000    3938 151  
  225   17.6  9.8  0.0  NODE_74_length_2645_cov_2.382625     815   865 (1780) + LINE-1            Unspecified    3427   3482  (1338) 152  
  228   12.5 25.0  0.0  NODE_750_length_887_cov_2.235577     768   823   (64) C BEL-22_DTa-I      Unspecified  (3264)   4849    4780 153  
  236   29.1 15.1  0.0  NODE_75_length_2642_cov_2.406262     371   456 (2186) C Gypsy-30_OD-I     Unspecified  (5464)   1113    1015 154  
  227   18.8  2.1  0.0  NODE_782_length_864_cov_1.948084     634   681  (183) + Chapaev-1N1_NV    Unspecified     535    583  (2001) 155  
  240   19.5  0.0  2.4  NODE_78_length_2622_cov_2.156213     160   201 (2421) + SART-2_PXu        Unspecified    3020   3060  (2380) 156  
  267   24.3  1.3  2.7  NODE_795_length_857_cov_1.842893     761   836   (21) + Gypsy-6_Ano-I     Unspecified     378    452  (4892) 157  
  235   20.0  1.5  4.5  NODE_7_length_5281_cov_3.893035      117   184 (5097) + Gypsy-18_AA-I     Unspecified    2860   2925  (1460) 158  
  227   18.2  6.8  0.0  NODE_7_length_5281_cov_3.893035     3464  3507 (1774) + Mariner-5_PTrit   Unspecified     622    668  (2915) 159  
  233   21.9  0.0  0.0  NODE_7_length_5281_cov_3.893035     4309  4349  (932) + Gypsy-1_CMe-I     Unspecified   10215  10255  (2155) 160  
  234   18.5 14.8  0.0  NODE_828_length_834_cov_2.192555     170   223  (611) C BEL-617_AA-I      Unspecified  (2199)   3649    3588 161  
  235   26.4  2.7  2.7  NODE_82_length_2557_cov_1.962430    1884  1957  (600) + Gypsy-14_DAn-I    Unspecified       6     79  (6026) 162 *
  312   27.4  0.0  0.0  NODE_82_length_2557_cov_1.962430    1892  1964  (593) C tRNA-Arg2_OS      Unspecified     (0)     73       1 163  
  246   16.3  2.3  0.0  NODE_835_length_831_cov_1.603093     109   151  (680) + EnSpm-8_CCri      Unspecified   10881  10924   (765) 164  
  239   25.0  0.0  0.0  NODE_852_length_820_cov_2.950327     242   297  (523) C ManaSINE1         Unspecified   (261)     70      15 165  
  256   23.7  0.0  1.7  NODE_852_length_820_cov_2.950327     334   393  (427) C tRNA-Asn_OS       Unspecified    (14)     66       8 166  
  227    7.5  4.3  2.9  NODE_85_length_2529_cov_2.308812      69   137 (2392) C TguERVL1a_LTR     Unspecified   (421)    256     187 167 *
  228   11.1  0.0  1.9  NODE_85_length_2529_cov_2.308812      83   137 (2392) C Harbinger-8_CCri  Unspecified  (2830)   2574    2521 168  
  225   11.5  0.0 11.4  NODE_86_length_2513_cov_2.644020    2344  2392  (121) + DNA8-1_CQ         Unspecified     154    197  (1960) 169  
  235   21.6  1.2 11.8  NODE_890_length_802_cov_2.016064     648   731   (71) + hAT-51_HM         Unspecified    3549   3624   (210) 170 *
  257   19.8  3.6  3.6  NODE_890_length_802_cov_2.016064     670   753   (49) + Ves2B_ML          Unspecified       7     90   (162) 171 *
  317   19.0  0.0  0.0  NODE_890_length_802_cov_2.016064     675   732   (70) + SINE2-2B_SSa      Unspecified       9     66   (116) 172  
  226   29.5  1.3  0.0  NODE_89_length_2482_cov_2.310260    2321  2398   (84) C BEL-2_LMi-I       Unspecified   (927)   4938    4860 173 *
  273   20.8  5.7  6.9  NODE_89_length_2482_cov_2.310260    2331  2418   (64) + BEL-2_LMi-I       Unspecified    5223   5309   (556) 174  
  233   14.3  8.3  6.6  NODE_968_length_739_cov_2.596491     184   243  (496) C Helitron-3_ALy    Unspecified  (8835)   6161    6101 175  
  243   17.8  0.0  0.0  NODE_96_length_2400_cov_2.190192    2157  2201  (199) + Helitron-1_Crp    Unspecified    4193   4237   (959) 176  
  234   25.8  0.0  4.5  NODE_973_length_736_cov_2.249633     239   307  (429) + MarinerL-1_CR     Unspecified    3248   3313 (12675) 177  
